public/downloads.php

```diff
@@ -26,6 +26,10 @@
   <body>
     <div class="top-row">
       <div class="container">
+        <div id="oldVersionWarning" class="oldVersionWarning">
+          You are trying to download builds for old/unsupported version of Minecraft!<br />
+          Keep in mind that if you download these builds, you won't get any support from Purpur!
+        </div>
```

Comment:
The old version warning was placed underneath the tabs/dropdown so that the tabs/dropdown don't move away from your cursor when you click on a version. If you can't find an alternative way to stop that from happening, the warning should stay below the dropdown/tabs.

Reply:
i will experiment with it later

Reply:
but you can push it now

Reply:
There'd be no point in requesting changes then lol

I'll merge once the version warning has been moved back or the issue I've outlined stops happening.

Reply:
sorry i forgot about it, i changed it back